lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -22,52 +22,63 @@
         description: location of the password store
         default: '~/.password-store'
       directory:
-        description: directory of the password store
+        description: The directory of the password store.
         env:
           - name: PASSWORD_STORE_DIR
       create:
-        description: flag to create the password
+        description: Create the password if it does not already exist.
         type: bool
         default: 'no'
       overwrite:
-        description: flag to overwrite the password
+        description: Override the password if it does already exist.
```

Comment:
Should be "Overwrite" rather than "override"

Reply:
Fixed